packages/daimo-api/src/server/pushNotifier.ts

```diff
@@ -123,15 +122,11 @@ export class PushNotifier {
         continue;
       }
 
-      const nonceMetadata = this.opIndexer.fetchNonceMetadata(
-        log.transactionHash,
-        log.logIndex
-      );
-
-      const receivingRequestedMoney =
```

Comment:
since nonces need rearchitecture soon anyway #585  we'll stop using request v1 in notifs